src/commands/ext-configure.ts

```diff
@@ -46,6 +47,9 @@ export default new Command("ext:configure <extensionInstanceId>")
   .before(diagnoseAndFixProject)
   .action(async (instanceId: string, options: Options) => {
     const projectId = getProjectId(options);
+    if (!projectId) {
```

Comment:
This isn;t true - you can run `firebase ext:configure --local` without a project id to reconfigure an extension in the manifest